packages/api/test/pin.spec.js

```diff
@@ -550,7 +557,7 @@ describe('Pinning APIs endpoints', () => {
     })
 
     it('returns the pin request', async () => {
-      const sourceCid = 'QmVGv1UK8EvhD9KMHdKBB1LWadiYai6sY5GHL6h7MHRWzf'
+      const sourceCid = 'bafybeidhbtemubjbsbuhyai5oaebqf2fdrvhnshbkncyqpnoy2bl2mpt4q'
```

Comment:
@LeslieOA I had to change this for my test to pass. Not sure if it's coming from the cid normalisation work you did? cc: @flea89

Reply:
@alexandrastoica Thanks for checking! It's from [Fix/psa unpinned status #933](https://github.com/web3-storage/web3.storage/pull/933) methinks. CC @flea89